techscreenr-angular/models/question-type.enum.ts

```diff
@@ -0,0 +1,6 @@
+export enum QuestionTypeEnum {
+    'Checkbox',
+    'Binary',
+    'FillInTheBlank',
+    'CodeExample'
```

Comment:
@amartinelli12  Consider removing "CodeExample" as an option because every question should in theory be able to have code as an example. Also, there needs to be a "radio" question option.